docs/en-US/component/switch.md

```diff
@@ -115,7 +115,7 @@ switch/custom-action-icon
 | before-change                 | before-change hook before the switch state changes. If `false` is returned or a `Promise` is returned and then is rejected, will stop switching | ^[boolean] / ^[Function]`() => Promise<boolean>` | —       |
 | id                            | id for input                                                                                                                                    | ^[string]                                        | —       |
 | tabindex                      | tabindex for input                                                                                                                              | ^[string] / ^[number]                            | —       |
-| label ^(a11y)                 | same as `aria-label` in native input                                                                                                            | ^[string]                                        | Switch  |
+| label ^(2.4.0) ^(a11y)        | same as `aria-label` in native input                                                                                                            | ^[string]                                        | —       |
```

Comment:
```suggestion
| label ^(2.4.1) ^(a11y)        | same as `aria-label` in native input                                                                                                            | ^[string]                                        | —       |
```

Sorry, 2.4.0 has already been released, we can only wait for the next release now